anti-tracking-threat-model.md

```diff
@@ -0,0 +1,196 @@
+# Anti-tracking Threat Model
+
+WebKit and other browsers are trying to reduce websites' ability to track users
+across the internet. While the success of their effort remains to be seen, we
+don't want Web Packaging to make the problem any more difficult.
+
+This document contains a proposed threat model for the best-case outcome of that anti-tracking effort. We do not yet claim there is consensus about any of:
+
+1. Whether the [attacker capabilities](#attacker-capabilities) are plausible.
+1. Whether it will eventually be possible to frustrate the [attacker goals that
+   we want to frustrate](#attacker-goals-that-we-want-to-frustrate).
+1. Whether the mitigations actually mitigate the attacks.
+1. Whether the costs of the mitigations are worth the benefits.
+1. Probably other things.
+
+Nonetheless, we feel it's important to start with a concrete proposal in mind
+when discussing how to evolve both this threat model and the Web Packaging
+proposals.
+
+Often when we criticize new, technically distinct tracking vectors, we are told
+that “you can track users in so many ways so why care about this one?” In the
+case of signed packages we hear about means of tracking such as doctored links
+where cross-site tracking is built into the URL, or server-side exchanges of
+personally identifiable information such as users' email addresses.
+
+Browsers are working hard to prevent cross-site tracking, including with new
+limits and restrictions on old technologies. Web Packaging must not add to that work, although it's acceptable for Web Packaging to prevent tracking only after that other work has been done.
```

Comment:
I don't think "that other work" is defined here. The other work is potentially years into the future and I don't think we should indicate that Web Packaging can punt on tracking issues in wait for such "other work."